src/krux/pages/home_pages/home.py

```diff
@@ -163,6 +163,14 @@ def bip85(self):
         bip85.export()
         return MENU_CONTINUE
 
+    def mnemonic_xor(self):
+        """Handler for the 'Mnemonic XOR' menu item"""
+        from .mnemonic_xor import MnemonicXOR
+
+        mnemonic_xor = MnemonicXOR(self.ctx)
+        mnemonic_xor.export()
```

Comment:
Result from XOR will be loaded, not exported, right? I think `export()` is not appropriated name for the function.